etc-conf/dbus/system.d/com.redhat.SubscriptionManager.conf

```diff
@@ -8,12 +8,35 @@
   <policy user="root">
     <allow own="com.redhat.SubscriptionManager"/>
     <allow own="com.redhat.SubscriptionManager.PluginEvent"/>
+
+    <!-- Only root can trigger methods in these services -->
+    <allow send_destination="com.redhat.SubscriptionManager"/>
+    <allow send_destination="com.redhat.SubscriptionManager.PluginEvent"/>
+
+    <!-- Basic D-Bus API stuff -->
+    <allow send_destination="com.redhat.SubscriptionManager"
+        send_interface="org.freedesktop.DBus.Introspectable"/>
+    <allow send_destination="com.redhat.SubscriptionManager"
+        send_interface="org.freedesktop.DBus.Properties"/>
+    <allow send_destination="com.redhat.SubscriptionManager"
+        send_interface="org.freedesktop.DBus.ObjectManager"/>
   </policy>
 
   <policy context="default">
-    <allow send_destination="com.redhat.SubscriptionManager"/>
-    <allow send_destination="com.redhat.SubscriptionManager.EntitlementStatus"/>
+    <!-- Common user can only check status of entitlement -->
+    <allow send_destination="com.redhat.SubscriptionManager"
+        send_path="/EntitlementStatus"
+        send_member="check_status"/>
+
+    <!-- Basic D-Bus API stuff -->
+    <allow send_destination="com.redhat.SubscriptionManager"
+        send_interface="org.freedesktop.DBus.Introspectable"/>
+    <allow send_destination="com.redhat.SubscriptionManager"
+        send_interface="org.freedesktop.DBus.Properties"/>
```

Comment:
I don't want to allow "org.freedesktop.DBus.Properties" by default, as this allows both getting and setting properties, and I don't want to need to come back in later and make it more restrictive if we do actually implement org.freedesktop.DBus.Properties... I'd like to just remove this allow statement for now. Please try this and ensure the fix still allows access to entitlement status as intended.